debian/control

```diff
@@ -105,3 +106,51 @@ Depends: eos-knowledge-0 (= ${binary:Version}),
          ruby-sass,
          ${misc:Depends}
 Description: EndlessOS Knowledge Library Devel package
+
+Package: libeos-knowledge-content-0-0
+Section: non-free/libs
+Architecture: any
+Depends: ${shlibs:Depends},
+         ${misc:Depends},
+         libglib2.0-bin,
+         libeos-shard-0-0 (>= 0.1.2),
+         gvfs
+Description: Endless Knowledge Content library
+ A library for serving up knowledge application offline content.
+
+Package: libeos-knowledge-content-0-dbg
+Section: non-free/debug
+Architecture: any
+Depends: ${misc:Depends},
+         libeos-knowledge-content-0-0 (= ${binary:Version})
+Description: Endless Knowledge Content library debugging symbols
+ Debugging symbols for the Endless Knowledge Content library
+
+Package: libeos-knowledge-content-0-dev
+Section: non-free/libdevel
+Architecture: any
+Depends: ${shlibs:Depends},
+         ${misc:Depends},
+         libglib2.0-dev,
+         libeos-knowledge-content-0-0 (= ${binary:Version})
+Description: Endless Knowledge Content development files
+ Package config and headers for the Endless Knowledge Content library
+
+Package: libeos-knowledge-content-0-doc
+Section: non-free/doc
+Architecture: any
+Depends: ${misc:Depends}
+Description: Endless Knowledge Content library documentation
+ HTML documentation for the Endless Knowledge Content library
+
+Package: gir1.2-eos-knowledge-content-0
```

Comment:
This should probably be `gir1.2-eosknowledgecontent-0` to fit with other upstream names, like `gir1.2-gdkpixbuf-2.0` (I know we named `gir1.2-eos-shard-0` incorrectly)